codespell_lib/data/dictionary.txt

```diff
@@ -19874,7 +19879,7 @@ quroum->quorum
 qutie->quite, quiet,
 rabinnical->rabbinical
 racaus->raucous
-ractise->practise
+ractise->practice
```

Comment:
Please undo this change as above, although you could add `ractice->practice` too